modules/apps/commerce/commerce-site-initializer/commerce-site-initializer-extender/src/main/java/com/liferay/commerce/site/initializer/extender/CommerceSiteInitializerImpl.java

```diff
@@ -770,7 +784,16 @@ private Channel _addOrUpdateCommerceChannel(
 
 		JSONObject jsonObject = _jsonFactory.createJSONObject(json);
 
-		jsonObject.put("siteGroupId", serviceContext.getScopeGroupId());
+		Group group = _groupLocalService.getGroup(
+			serviceContext.getScopeGroupId());
+
+		String siteName = group.getName(serviceContext.getLanguageId());
+
+		if (Validator.isNull(jsonObject.getString("name"))) {
+			jsonObject.put("name", siteName);
+		}
+
+		jsonObject.put("siteGroupId", group.getGroupId());
```

Comment:
same here,
We can also move the replace methods to live under SiteInitializerUtil, allowing them to be called in all SiteInitializers